src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json

```diff
@@ -356,7 +356,15 @@
     },
     "RequiresHttps": {
       "type": "computed",
-      "value": "(OrganizationalAuth || IndividualB2C || !NoHttps)"
+      "value": "(OrganizationalAuth || IndividualB2CAuth)"
+    },
+    "SupportsHttp": {
```

Comment:
nit: Should we name the this Define/UseHttp[s]Profile instead of Supports since this is what controls the launch settings having the profile?

Reply:
Yeah I had named it that way originally, but it's also used to gate the inclusion of the hsts/https middleware in `Program.cs`. Still, it might be clearer named something like `HasHttpsProfile`/`HasHttpProfile`.

Reply:
Yeah I clearer is better for this stuff, HasHttpsProfile/HasHttpProfile sounds good